src/components/Input/DropDown/DropDownGroup.js

```diff
@@ -85,6 +86,14 @@ class DropDownGroup extends React.Component {
         this.toggleDropdown();
         break;
       default:
+        if (!isOpen) {
+          const { word } = this.state;
+          if (keyCode > 48 && keyCode < 90) {
+            const temp = word + e.key;
+            this.setState({ word: temp }, () => this.searchValue(temp));
+            this.cleanWord();
```

Comment:
Do I undetstand correctly, that search field will be cleared in a second?

Reply:
There is no visible search field, so kinda. When the dropdown is focused and the user starts and stops writing something, the component will forget, what the user wrote after 1 sec. Tried to emulate the native behavior.